### PR TITLE
Add ING (NL)

### PR DESCRIPTION
**Pull request by pitdicker**
The record for ING is mostly similar to their branches in other countries.
I also updated the logo the the one that is used as favicon and app logo.
The available SVG was already compressed a too far by a tool like svgomg, so I didn't run it again (not a fan anyway).

I am not sure about the current state of sms support. I believe it is necessary to have a verified number, among other things, but that it is no longer used for authentication.

### CODE REVIEW

File: entries/i/ing.nl.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "ING": {
```
Comment (Carlgo11) [inline entries/i/ing.nl.json:2]:
```suggestion
  "ING (NL)": {
```

Comment (phallobst) [inline entries/i/ing.nl.json:2]: This should include some country idenfication, e.g. `ING (NL)`, `ING (Netherlands)` or `ING (Nederland)`.

**Review by phallobst — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
The logo provided looks a bit too compressed.
I recommend one of these two:
Square ING logo (4.9KiB):
square icon

Rounded logo (3.3KiB):
rounded icon

**Comment by Carlgo11** [comment]
As the requested changes haven't been resolved yet I'm going to label this PR as inactive.
Unless there's activity within the next week I'll close the PR.

**Comment by Carlgo11** [comment]
Closing due to inactivity.

**Pull request closed without merging**